alinka/widget/actions.py

```diff
@@ -3,7 +3,7 @@
 from alinka.schemas import DecisionDbSchema, DocumentData
 
 
-def generate_and_save_decision(form_data: DocumentData, generate: bool = False) -> None:
+def generate_and_save_decision(form_data: DocumentData, generate: bool = False, destination_path: str = "") -> None:
```

Comment:
what will happen, when indeed somebody will pass `""` to `destination_path` (or when we will rely on default)? Do we need that default?